frontend/src/core/models/board/task-Info-model.ts

```diff
@@ -0,0 +1,4 @@
+export interface TaskInfoModel {
+    name: string
+    avatarUrl: string
```

Comment:
Why does task have avatarUrl?

Reply:
This is a fake model, the person who will make the card will replace it with the necessary fields